src/Transports.AspNetCore/Common/IGraphQLRequestDeserializer.cs

```diff
@@ -0,0 +1,15 @@
+using GraphQL.Server.Common;
+using Microsoft.AspNetCore.Http;
+using System.Threading.Tasks;
+
+namespace GraphQL.Server.Transports.AspNetCore.Common
+{
+    public interface IGraphQLRequestDeserializer
+    {
+        Task<GraphQLRequestDeserializationResult> DeserializeAsync(HttpRequest httpRequest);
```

Comment:
Maybe it’s worth writing here that request baching is supported only for the request body.

Reply:
Done. Documented the whole interface a whole lot better.